innopoints/views/account.py

```diff
@@ -157,6 +157,12 @@ def get_timeline(email):
             .join(Project,
                   Project.id == Notification.payload.op('->>')('project_id').cast(db.Integer))
             .add_column(Project.name.label('project_name'))
+            .outerjoin(Activity, (Activity.project_id == Project.id)
+                               & (Activity.internal)
+                               & (Activity.name == 'Moderation'))
```

Comment:
So you are using the name as a filter after all?

Reply:
Yes, with the intention that there might be other internal activities

Reply:
What other kinds of internal activities could there possibly be?

Reply:
Doesn't matter. Let's just make room, it's not like this feature is a hit on performance or code quality